web/multinode/package.json

```diff
@@ -16,6 +16,7 @@
     "vue-chartjs": "3.5.1",
     "vue-class-component": "7.2.6",
     "vue-clipboard2": "0.3.1",
+    "vue-notification": "^1.3.20",
```

Comment:
@mobyvb don't forget to review the dependency. https://github.com/storj/docs/blob/main/code/Dependencies.md

Reply:
Also don't we already have some sort of notification system?

Reply:
Thank you @egonelbre 
I was planning on reviewing the dependency before approving.

We do have a custom notification system for the satellite web app. @VitaliiShpital has shared the following with me about how it is written/implemented:

> ok, so it consists of explicit vuex module https://github.com/storj/storj/blob/main/web/satellite/src/store/modules/notifications.ts
notificator plugin https://github.com/storj/storj/blob/main/web/satellite/src/utils/plugins/notificator.ts
which is enabled in main.ts like this https://github.com/storj/storj/blob/main/web/satellite/src/main.ts#L21
here we add it to UI https://github.com/storj/storj/blob/main/web/satellite/src/App.vue#L8
and here’s an example of it’s usage https://github.com/storj/storj/blob/main/web/satellite/src/components/accessGrants/AccessGrants.vue#L119

I still need to audit the vue-notification package (link for me, later: https://github.com/euvl/vue-notification).

Reply:
I have reviewed the `vue-notification` package, and it seems fine. It does not require any additional dependencies besides itself, and I didn't notice any issues code in the code (there is not very much code).

@Pranav2612000 would you like me to review this PR as-is, or do you want to try implementing notifications like `web/satellite`, which I linked in my previous comment?

Reply:
> I have reviewed the vue-notification package, and it seems fine. It does not require any additional dependencies besides itself, and I didn't notice any issues in the code (there is not very much code).

That does not seem to be the case. There seem to be plenty of additions in the lock file, e.g. `fsevents`, `node-notifier`, `fork-ts-checker-webpack-plugin-v5`, `vue-loader-v16`, `prettier`, `source-map`. Did you compare the before and after of the node_modules folder?

And if we already have a system for notifications then we should diverge only if we have a really good reason and/or plan to move other implementations to it.

Reply:
Oh, I said that because I was looking at the package.json here https://github.com/euvl/vue-notification/blob/master/package.json and did not see any non-dev dependencies added.
But yes, I see that the package lock has been updated. And I have not audited any of those new packages other than vue-notification.

Reply:
0kay. Does that mean we should use the notification system already being used? Can you point me to the documentation for it? I'll be happy to use it. ( I had also asked if we have a notification system before, but I got to know we don't have one and that's why I used this package )

Reply:
@mobyvb the notification system we already have is only in the satellite so how do we share it with the web/storagenode and web/multinode ?